perceval/backends/core/discourse.py

```diff
@@ -299,27 +308,31 @@ class DiscourseClient(HttpClient):
     TOPIC = 't'
     POSTS = 'posts'
 
+    # Headers
+    HKEY = 'Api-Key'
+    HUSER = 'Api-Username'
+
     # Params
-    PKEY = 'api_key'
     PPAGE = 'page'
 
     # Data type
     TJSON = '.json'
 
-    def __init__(self, base_url, api_key=None, sleep_time=DEFAULT_SLEEP_TIME, max_retries=MAX_RETRIES,
+    def __init__(self, base_url, api_username=None, api_key=None, sleep_time=DEFAULT_SLEEP_TIME, max_retries=MAX_RETRIES,
```

Comment:
Would you mind to make this line shorter? Before the change was also lager than it should, in my opinion.